ogre_mesh_viewer.py

```diff
@@ -200,6 +201,21 @@ def draw_loading(self):
         ImGui.Text("Loading..            ")
         ImGui.End()
 
+    def reload(self):
+        if app.infile:
+            app.restart = True
+            app.getRoot().queueEndRendering()
+
+    def load_file(self):
+        # Avoid recursive calling, which might block the window manager
+        if self.load_file_lock:
+            return
+        else:
+            self.load_file_lock = True
+        app.infile = askopenfilename(app.filedir)
+        self.load_file_lock = False
+        self.reload()
```

Comment:
```suggestion
        app.reload()
```

logic should be in the app, not the gui (Controller-View separation)